include/mbedtls/md.h

```diff
@@ -49,6 +49,12 @@ typedef enum {
     MBEDTLS_MD_RIPEMD160,
 } mbedtls_md_type_t;
 
+typedef enum {
+    MBEDTLS_SIG_RSA=0,
+    MBEDTLS_SIG_ECDSA,
+    MBEDTLS_SIG_NONE,
```

Comment:
NONE should be the first value and equal to 0:
- it's a better default value to be used when structures that contain a member of this type are zeroed (as a result of `calloc()` for example)
- if we ever add value to that enum, we can't change the position of existing values (for ABI compatibility), and it would look weird to have NONE in the middle of the list.